azurerm/internal/services/network/tests/resource_arm_subnet_test.go

```diff
@@ -442,6 +461,27 @@ resource "azurerm_subnet" "test" {
 `, template, enabled)
 }
 
+func testAccAzureRMSubnet_basic_addressPrefixes(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
```

Comment:
```suggestion
  name     = "acctestRG-n-%d"
```
Updating for best practices